content/challenges/easy/6 - Flip the Boolean/func.js

```diff
@@ -5,6 +5,10 @@
  *
  * @returns {boolean|string}
  */
-module.exports = (bool) => {
-    // your code here
-};
+ module.exports = (bool) => {
+ if (typeof bool === "boolean") {
+    return !bool;
+ } else {
+    return "boolean expected";
+ }
+ };
```

Comment:
Indent needed here. Its hard to tell if the if statement is a sibling or a child of `module.exports = .....` (hint: it should appear as a child in that it's contained inside of

```
parentCode {
  childCode = ...;
}
```

Yours looks like this
```
parentCode{
parentCode; //but is this really a parent or is it contained inside of parentCode and is actually inside that block?
}
```